common/lib/protocol-definitions/CHANGELOG.md

```diff
@@ -1,5 +1,13 @@
 # @fluidframework/protocol-definitions Changelog
 
+## [2.0.0]
+
+### Remove `MessageType.RemoteHelp` from protocol definitions
+
+The `RemoteHelp` operation type has been removed from `protocol-definitions`. It has not been in use for
+a long time and is being removed in this release.
+
```

Comment:
@tylerbutler @ChumpChief as the last people to have touched this file, what's the process for creating the tag and that I need to populate this readme with? I see you created a tag for the last release and put it into this readme.

Reply:
It got a little more complicated than it needed to be. @ChumpChief made a "breaking change" in protocol defs, but we need to release a minor version with a deprecation first. So the deprecation went to main, but it looks like the package was never released at 1.2.0. I'll release it later today, but is there anything you want to add to it, like deprecating what you're removing here?

Reply:
Once 1.2.0 is out, you can do one of two things:

1. Make your breaking change in main and bump the version to 2.0, then release it and integrate it into the relevant places. Once done bump the version in next to 3.0.
2. Make your breaking change in next and wait for the next client major release for the changes to be available for integration.

Reply:
Re: my change to protocol-defs - I don't need an on-demand release, feel free to defer to whenever we would normally release a new minor.  Waiting for any deprecations from Kajari's change seems OK from my side.